src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -298,12 +303,70 @@ def _start_cinema4d_client(self) -> None:
             new_module_path = plugin_dir + os.pathsep + module_path
         os.environ[module_path_key] = new_module_path
 
+        arguments = [c4d_exe, "-nogui", "-DeadlineCloudClient"]
+        if "linux" in platform.system().lower():
+            _logger.info("Setting Linux Cinema4D environment")
+            env = self._get_cinema4d_environment(c4d_exe)
+            _logger.info("Inserting Linux adaptor wrapper script")
+            arguments.insert(0, os.path.join(os.path.dirname(__file__), "adaptor.sh"))
+
+        if os.environ.get("g_licenseServerRLM", None):
+            arguments.append("g_licenseServerRLM={}".format(os.environ.get("g_licenseServerRLM")))
+        if os.environ.get("g_licenseServerUrl", None):
+            arguments.append("g_licenseServerUrl={}".format(os.environ.get("g_licenseServerUrl")))
+
```

Comment:
Hmm, is this required in Linux environments? I set this environment in Windows and I didn't need to pass it as an argument to commandline during my tests.

Reply:
yea seems a little odd to me that we need to add it in both spots?